DoLah/Parser/CommandParser.cpp

```diff
@@ -155,6 +155,7 @@ namespace DoLah {
 
 
     AbstractCommand* CommandParser::parse(std::string input) {
+        input = ParserLibrary::removeDoubleSpacing(input);
```

Comment:
will simply ignore it.

Quite a lot of software behave in similar manner I believe.